var/spack/repos/builtin/packages/py-openai-gym/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyOpenaiGym(PythonPackage):
+    """OpenAI Gym is a toolkit for developing and comparing
+    reinforcement learning algorithms. This is the gym open-source
+    library, which gives you access to a standardized set of
+    environments."""
+
+    homepage = "https://github.com/openai/gym"
+    url      = "https://github.com/openai/gym/archive/refs/tags/0.18.1.tar.gz"
```

Comment:
Can we download from PyPI instead? It doesn't look like 0.18.1 has made it to PyPI yet, but 0.18.0 is there.